core/services/ocr2/plugins/rebalancer/bridge/bridge.go

```diff
@@ -0,0 +1,42 @@
+package bridge
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/rebalancer/models"
+)
+
+type Bridge interface {
+	// PopulateStatusOfTransfers determines the transfer status and returns the transfers
+	// with all the statuses being set.
+	PopulateStatusOfTransfers(ctx context.Context, token, sender common.Address, transfers []models.Transfer) ([]models.PendingTransfer, error)
```

Comment:
Can we use `models.Address` here instead since we should strive to be chain-agnostic in these interfaces?

```suggestion
	PopulateStatusOfTransfers(ctx context.Context, token, sender models.Address, transfers []models.Transfer) ([]models.PendingTransfer, error)
```

Feel free to add helper methods to `models.Address` to return an EVM address if the implementation of the bridge is definitely EVM.

Reply:
oops good catch